### PR TITLE
[workloads] Fix the condition for the workload metrics

### PR DESCRIPTION
**Pull request by lewing**

### CODE REVIEW

**Comment by lewing** [comment]
should backport too

File: src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.targets.in [hunk 1]
```diff
@@ -239,6 +239,6 @@
         Importance="High" />
   </Target>
 
-  <Import Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '8.0')) and ('$(UsingBrowserRuntimeWorkload)' == 'true' or '$(UsingMobileWorkload)' == 'true')"
+  <Import Condition="'$(_MonoWorkloadTargetsMobile)' == 'true'"
```
Comment (radical) [inline src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.targets.in:242]: Should the condition here be updated to match for `>8.0`?
https://github.com/dotnet/runtime/blob/542983327f61706b2ed07583601861846361364b/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.targets.in#L165-L169

This property seems to be true for 6/7/8 also, like: https://github.com/dotnet/runtime/blob/542983327f61706b2ed07583601861846361364b/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net7.Manifest/WorkloadManifest.targets.in#L60-L63

Comment (lewing) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.targets.in:242]: I think it works fine either way but @steveisok can decide

Comment (steveisok) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.targets.in:242]: I have no preference. In any case, we should try to be as consistent as possible.

**Review by steveisok — APPROVED** [review]

**Pull request merged**